### PR TITLE
python-modernize . -w -f libmodernize.fixes.fix_input_six

### PR DESCRIPTION
**Pull request by nickpell**
@dimagi/py3

### CODE REVIEW

File: scripts/couchdb/utils.py [hunk 1]
```diff
@@ -34,7 +35,7 @@ def get_membership(node_details):
 
 
 def confirm(msg):
-    return raw_input(msg + "\n(y/n)") == 'y'
+    return input(msg + "\n(y/n)") == 'y'
```
Comment (emord) [inline scripts/couchdb/utils.py]: @dannyroberts should scripts/couchdb get removed from HQ since you created that other repo?

**Review by emord — APPROVED** [review]
not familiar with the input vs raw_input changes on py3, but this is only mgmt commands so i'm not going to worry about it

**Review by emord — APPROVED** [review]

File: scripts/couchdb/utils.py (hunk 1 above)
Comment (dannyroberts) [reply scripts/couchdb/utils.py]: Oh. Yeah, totally. Man I've been forgetting things like it's my job. I'll go clean that up

**Comment by dannyroberts** [comment]
> not familiar with the input vs raw_input changes on py3, but this is only mgmt commands so i'm not going to worry about it

I believe in python2 `input` is basically `lambda msg: eval(raw_input(msg))` or something horrible like that. They changed it in python3 so that the innocuous-sounding `input` function just receives user input and doesn't also evaluate it as python. Since this is what `raw_input` does in python2, that basically means `input` => ø and `raw_input` => `input`.

File: scripts/couchdb/utils.py (hunk 1 above)
Comment (dannyroberts) [reply scripts/couchdb/utils.py]: https://github.com/dimagi/commcare-hq/pull/18498

**Pull request merged**